b2share/modules/deposit/api.py

```diff
@@ -361,7 +361,8 @@ def create_file_pids(record_metadata):
                            bucket_id=f.get('bucket'), key=f.get('key'),
                            _external=True)
         try:
-            file_pid = createHandle(file_url, checksum=f.get('checksum'))
+            file_pid = create_handle(file_url, checksum=f.get('checksum'),
```

Comment:
Same as records. At some point we should register the HANDLE PIDs after the commit in an asynchronous task. Otherwise we will have from time to time PIDs which are registered but the transaction rollbacks because of some error.